__tests__/select.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import Component from '../src/select';
+import snapshot from './snapshot';
+
+describe(Component.name, () => {
+  const onChange = jest.fn();
+  const options = [
+    {text: 'A', value: 'a'},
+    {text: 'B', value: 'b'},
+    {text: 'C', value: 'c'}
+  ];
+  const size = 5;
+  const value = 'hello';
+  const getDefaultProps = () => ({onChange, options, size, value});
+
+  it('should render', () => {
+    snapshot(<Component {...getDefaultProps()} />);
+  });
+
+  it('should have a className of .select on the select element', () => {
```

Comment:
I think the snapshot test above covers all of the tests except the one that calls `simulate`.